docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with PyTorch and Cleanlab\n",
+    "\n",
+    "This quickstart tutorial demonstrates how to find issues in image classification data. Here we use the Fashion-MNIST dataset containing 70,000 images of fashion products from 10 categories, with 7,000 images per category.\n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Build a simple [PyTorch](https://pytorch.org/) neural net.\n",
+    "\n",
+    "- Use cross validation to compute out-of-sample predicted probabilities (`pred_probs`) and feature embeddings (`features`).\n",
+    "\n",
+    "- Utilize these `pred_probs` and `features` to identify potential issues within the dataset using the `Datalab` class from cleanlab. The issues found by cleanlab include mislabeled examples, near duplicates, outliers, and image-specific problems such as excessively dark or low information images."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have a `model`? Run cross-validation to get out-of-sample `pred_probs` and then the code below to find any potential label errors in your dataset.\n",
```

Comment:
why is this quick-start block only finding label issues? It should find all issues like the main datalab quickstart block

Reply:
Changed